siriuspy/siriuspy/clientconfigdb/types/as_diagnostics.py

```diff
@@ -414,6 +414,14 @@ def get_dict():
     [':SwInvGainB-SP', 1.0, 0.0],
     [':SwInvGainC-SP', 1.0, 0.0],
     [':SwInvGainD-SP', 1.0, 0.0],
+    [':GEN_PolyXArrayCoeff-SP', 1.0, 0.0],
```

Comment:
Considering these PVs are arrays, shouldn't you set the default value as an array? maybe the default value could be the array we use today?

Reply:
done!